lib/api/src/grpc/proto/health_check.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
```

Comment:
I'd love a small comment pointing to the reference https://github.com/grpc/grpc/blob/master/doc/health-checking.md#service-definition

Reply:
Added the link as a comment.